lib/get-ajv-error-messages.js

```diff
@@ -0,0 +1,53 @@
+// see https://github.com/epoberezkin/ajv#validation-errors
+
+const getDetailsPerAjvKeyword = {
+  propertyNames(error, allErrors) {
+    let errorMessage = error.message;
+
+    const relatedError = allErrors.find(
+      err => err.propertyName === error.params.propertyName
+    );
+    if (relatedError) {
+      errorMessage += ` (${relatedError.message})`;
+    }
+
+    return errorMessage;
+  },
+  additionalProperties(error) {
+    return `${error.message} (${error.params.additionalProperty})`;
+  },
+  enum(error) {
+    return `(${error.data}) should be equal to one of [${error.params.allowedValues.join(`, `)}]`;
```

Comment:
Please also for patterns

Reply:
How exactly do you mean? Why did you comment at this line?

I think a similar error message would be possible in the `oneOf` keyword handler (given there are only schemas with only `pattern` in the `oneOf` array), but splitting one pattern into multiple sub-patterns at `|` isn't feasible.